Hive.NET.Core/Components/BeeWorkItem.cs

```diff
@@ -1,21 +1,33 @@
-﻿namespace Hive.NET.Core.Components;
+using System;
+using System.Threading.Tasks;
+
+namespace Hive.NET.Core.Components;
 
 public class BeeWorkItem
 {
-    public BeeWorkItem(Task task, Action onSuccess = default, Action<Exception> onFailure = default)
+    public BeeWorkItem(
+        Task task, 
+        string? description = null,
+        Action onSuccess = default, 
+        Action<Exception> onFailure = default)
```

Comment:
Double check if those literals can be converted to not nullable types

Reply:
only "description" as newly added as marked as nullable and Its not mandatory param so it can be null